Sonam_HW03.py

```diff
@@ -0,0 +1,112 @@
+#################################################################################
+# Author: [Your Name]
+# Username: [Your Username]
+#
+# Assignment: HW03
+# Purpose: Create a colorful house using functions.
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+import turtle
+
+
+def create_house(turtle_instance, size, house_color):
+    """
+    Constructs the main structure of the house.
+    """
+    turtle_instance.fillcolor(house_color)
+    turtle_instance.begin_fill()
+    for _ in range(4):
+        turtle_instance.forward(size)
+        turtle_instance.right(90)
+    turtle_instance.end_fill()
+
+
+def create_roof(turtle_instance, size, roof_color):
+    """
+    Constructs the roof of the house.
+    """
+    turtle_instance.fillcolor(roof_color)
+    turtle_instance.begin_fill()
+    for _ in range(3):
+        turtle_instance.forward(size)
+        turtle_instance.left(120)
+    turtle_instance.end_fill()
+
+
+def create_door(turtle_instance, door_width, door_height, door_color):
+    """
+    Constructs the door of the house.
+    """
+    turtle_instance.fillcolor(door_color)
+    turtle_instance.begin_fill()
+    for _ in range(2):
+        turtle_instance.forward(door_width)
+        turtle_instance.right(90)
+        turtle_instance.forward(door_height)
+        turtle_instance.right(90)
+    turtle_instance.end_fill()
+
+
+def create_window(turtle_instance, window_size, window_color):
+    """
+    Constructs a window for the house.
+    """
+    turtle_instance.fillcolor(window_color)
+    turtle_instance.begin_fill()
+    for _ in range(4):
+        turtle_instance.forward(window_size)
+        turtle_instance.right(90)
+    turtle_instance.end_fill()
+
+    turtle_instance.forward(window_size / 2)
+    turtle_instance.right(90)
+    turtle_instance.forward(window_size)
+    for _ in range(2):
+        turtle_instance.right(90)
+        turtle_instance.forward(window_size / 2)
+    turtle_instance.right(90)
+    turtle_instance.forward(window_size)
+
+
+def setup_scene():
```

Comment:
We use the name `main()` as a standard, even though it's not required. Convert it as such please.